app/src/main/sqldelight/ca/llamabagel/transpo/data/db/RecentStop.sq

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE recent_stop (
+    id TEXT NOT NULL UNIQUE PRIMARY KEY,
+    code TEXT NOT NULL,
+    name TEXT NOT NULL,
+    parent_station TEXT,
+    date INTEGER NOT NULL,
+    FOREIGN KEY (parent_station) REFERENCES stop(id) ON DELETE CASCADE
+);
+
+insert:
+REPLACE INTO recent_stop(id, code, name, parent_station, date)
+VALUES (:id, :code, :name, :parentStation, :date);
+
+selectMostRecent:
```

Comment:
use get terminology instead of select. The function will be consumed in Kotlin, not SQL.